added perscom plugin v2 docs (#3)
forumify/forumify-docs: src/app/plugins/perscom-plugin/introduction/page.mdx

Reviewer: Maybe include something about the initial sync here?

Users will see a big orange banner on all forumify PERSCOM pages if they have not synced yet. It might be a bit jarring if they see that after configuring their PERSCOM, and there being no mention of it anywhere in the docs.

Perhaps we can also have a section (or a separate page) just covering the sync in general. Like changes you do in forumify are immediately synced to PERSCOM.io in the background. While changes you do in PERSCOM.io are only synced during the night, unless you trigger a manual sync. We can include a screenshot of the sync page or something here (Admin Panel > PERSCOM > Sync).

Maybe the Users section can be replaced with this?

import GuideNavigator from '@/components/GuideNavigator';
import Alert from '@/components/Alert';
import { ArrowsLeftRight } from '@phosphor-icons/react/dist/ssr';

# Introduction

**Welcome to the documentation for Forumify's PERSCOM.io Integration Plugin**
Whether you're brand new to personnel management systems and forumify or just getting started, this guide will walk you through your
first set up in minutes.

This guide has been  broken up into sections, making it short and sweet for the readers to follow along.

# Installation
Seamlessly integrate PERSCOM.io into your forumify instance.

Package name for manual installation: `forumify/forumify-perscom-plugin`

> <Alert severity="warning">The PERSCOM.io plugin requires you to have PERSCOM.io PRO license or higher for API access.</Alert>

Reviewer: ```suggestion
<Alert severity="warning">The PERSCOM.io plugin requires you to have an active PERSCOM.io subscription for API access.</Alert>
```

PERSCOM.io doesn't have tiers anymore
